staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1589,6 +1589,12 @@ message ContainerEventResponse {
 
     // ID of the sandbox container
     PodSandboxMetadata pod_sandbox_metadata = 4;
```

Comment:
With the status added do we still need pod_sandbox_metadata?

Reply:
@ruiwen-zhao we end up using the function `determinePodSandboxIPs` from `kuberuntime_sandbox.go` at, 

https://github.com/kubernetes/kubernetes/pull/111384/files#diff-0e1a8d22b0f1dc42d412749c6a916b4c76394ead739d4bfbf26564faedd74a2fR968

which uses `Name` and `Namespace` fields (although only for logging)

Reply:
gotcha, yeah in this case I guess it should be fine to keep it, as long as kubelet doesn't need to get pod status after receiving the events.